clang/lib/Driver/Driver.cpp

```diff
@@ -9091,7 +9091,8 @@ InputInfoList Driver::BuildJobsForActionNoCache(
       // in this instance
       Action::OffloadKind DependentOffloadKind;
       if (UI.DependentOffloadKind == Action::OFK_SYCL &&
-          TargetDeviceOffloadKind == Action::OFK_None)
+          TargetDeviceOffloadKind == Action::OFK_None &&
+          !isSYCLNativeCPU(C.getDefaultToolChain().getTriple(), TC->getTriple()))
```

Comment:
Is there a test that can be updated to track the kind consistency?

Reply:
I don't think we have one, I'll add it to the PR

Reply:
@mdtoguchi I've updated the lit test for native cpu, I'm still looking into the CI failures in the driver lit tests